pyinfra/operations/files.py

```diff
@@ -1661,7 +1666,7 @@ def block(
         )
 
         # have two entries in /etc/host
-        files.marked_block(
+        files.block(
```

Comment:
#984 
[this comment](https://github.com/pyinfra-dev/pyinfra/issues/984#issuecomment-1565657899)